From 4f19837ccdeb12df52c415db6ab921f0d042efe3 Mon Sep 17 00:00:00 2001
From: wangjiezhe <wangjiezhe@gmail.com>
Date: Sun, 7 Jul 2024 16:04:17 +0800
Subject: [PATCH] Add Lenovo M7600D support

---
 README.md      | 1 +
 brlaser.drv.in | 9 +++++++++
 2 files changed, 10 insertions(+)

diff --git a/README.md b/README.md
index 3865baf..6c87692 100644
--- a/README.md
+++ b/README.md
@@ -84,6 +84,7 @@ The following printers have been reported to work with this driver:
 * Brother MFC-L2750DW series
 * Fuji Xerox DocuPrint P265 dw
 * Lenovo LJ2650DN
+* Lenovo M7600D
 
 
 Installation
diff --git a/brlaser.drv.in b/brlaser.drv.in
index db79006..30ddac9 100644
--- a/brlaser.drv.in
+++ b/brlaser.drv.in
@@ -634,3 +634,12 @@ Option "brlaserEconomode/Toner save mode" Boolean AnySetup 10
   Duplex rotated
   PCFileName "lnj2650d.ppd"
 }
+
+{
+  ModelName "M7600D"
+  Attribute "NickName" "" "Lenovo M7600D, $USING"
+  Attribute "1284DeviceID" "" "MFG:Lenovo;CMD:PJL,XL2HB;MDL:M7600D;CLS:PRINTER;"
+  Resolution k 1 0 0 0 "300dpi/300 DPI"
+  Duplex normal
+  PCFileName "m7600d.ppd"
+}
